### ISSUE TITLE
make deb Fails

### ISSUE THREAD

**Issue by dklann** [ElvishArtisan/rivendell#769]
Attempting to build on Ubuntu Studio 20.04 (up to date). For (at least) commit _5a42be85cc21d_ `make deb` fails with
```
./configure: line 3282: - : syntax error: operand expected (error token is "- ")
```
after:
```
make[1]: Entering directory '/home/dklann/src/rivendell/rivendell-4.0.0beta3'
dh_update_autotools_config
dh_autoreconf
```
I then see the following:
```
cat: versions/RIVWEBCAPI_CURRENT: No such file or directory
cat: versions/RIVWEBCAPI_REVISION: No such file or directory
cat: versions/RIVWEBCAPI_AGE: No such file or directory
cat: versions/RIVWEBCAPI_CURRENT: No such file or directory
cat: versions/RIVWEBCAPI_REVISION: No such file or directory
cat: versions/RIVWEBCAPI_AGE: No such file or directory
cat: versions/RIVWEBCAPI_CURRENT: No such file or directory
cat: versions/RIVWEBCAPI_REVISION: No such file or directory
cat: versions/RIVWEBCAPI_AGE: No such file or directory
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
cat: versions/RIVWEBCAPI_CURRENT: No such file or directory
cat: versions/RIVWEBCAPI_REVISION: No such file or directory
cat: versions/RIVWEBCAPI_AGE: No such file or directory
cat: versions/RIVWEBCAPI_CURRENT: No such file or directory
cat: versions/RIVWEBCAPI_REVISION: No such file or directory
cat: versions/RIVWEBCAPI_AGE: No such file or directory
cat: versions/RIVWEBCAPI_CURRENT: No such file or directory
cat: versions/RIVWEBCAPI_REVISION: No such file or directory
cat: versions/RIVWEBCAPI_AGE: No such file or directory
cat: versions/RIVWEBCAPI_CURRENT: No such file or directory
cat: versions/RIVWEBCAPI_REVISION: No such file or directory
cat: versions/RIVWEBCAPI_AGE: No such file or directory
```
I'm not sure how to track this down. The tangled web of Makefiles and configure scripts leaves my head spinning. Continuing to investigate. `make all` works as expected.

**Comment by dklann** [comment]
Another data point. Those three missing files in _versions_ are not in the tarball that gets created with `make deb` (_./rivendell_4.0.0beta3.orig.tar.gz_).

**Comment by dklann** [comment]
Found the issue. Pull request forthcoming...

**Comment by ElvishArtisan** [comment]
Confirmed here.

**Comment by ElvishArtisan** [comment]
Fixed by #770.
